### DISCUSSION TITLE
健康检查

### DISCUSSION THREAD

**Discussion by lovestudylalala** [apache/apisix#5489] [Q&A]
请问下如何可以查看到上游健康检查有问题的服务器的结果

**Comment by tokers** [comment]
See http://apisix.apache.org/docs/apisix/control-api/#get-v1healthcheck.

**Reply by zhangshunbao** [reply]
tengine的主动检查结果有自带UI的，apisix有计划加吗？
http://tengine.taobao.org/document_cn/http_upstream_check_cn.html

**Reply by juzhiyuan** [reply]
@zhangshunbao Could you please share the UI screenshot or demo?

**Reply by zhangshunbao** [reply]
1. nginx:     ngx_http_upstream_check_module (static)
2. location :
        location /status {
            check_status;

            access_log   off;
            allow SOME.IP.ADD.RESS;
            deny all;
        }
3. result url :
http://bbs.linuxtone.org/data/attachment/forum/201205/11/182048rfev3nnfjuzxxwdw.png
http://bbs.linuxtone.org/data/attachment/forum/201205/11/182048ph4drmprrxxourho.png

**Comment by lovestudylalala** [comment]
好滴，谢谢
